test/data/manifests/fedora-boot.mpp.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   generators:
     - name: osbuild-mpp
       version: unknown
+  exports:
+    - name: os
```

Comment:
Wouldn't we put `image` here?

Reply:
Yes, that makes more sense; I just took the first pipeline I saw :)